packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogSdkPlugin.kt

```diff
@@ -256,112 +281,88 @@ class DatadogSdkPlugin : FlutterPlugin, MethodCallHandler {
         }
         method?.let {
             it.isAccessible = true
             it.invoke(target)
         }
     }
 
-    internal fun stop() {
-        simpleInvokeOn("stop", Datadog)
-
-        val rumRegisteredField = GlobalRum::class.java.getDeclaredField("isRegistered")
-        rumRegisteredField.isAccessible = true
-        val isRegistered: AtomicBoolean = rumRegisteredField.get(null) as AtomicBoolean
-        isRegistered.set(false)
-    }
-
-    private fun attachEventMappers(
-        config: DatadogFlutterConfiguration,
-        configBuilder: Configuration.Builder
-    ) {
-        if (config.attachLogMapper) {
-            configBuilder.setLogEventMapper(
-                object : EventMapper<LogEvent> {
-                    override fun map(event: LogEvent): LogEvent? {
-                        return mapLogEvent(event)
-                    }
-                }
-            )
-        }
-
-        config.rumConfiguration?.let {
-            if (it.attachViewEventMapper) {
-                configBuilder.setRumViewEventMapper(
-                    object : ViewEventMapper {
-                        override fun map(event: ViewEvent): ViewEvent {
-                            return rumPlugin.mapViewEvent(event)
-                        }
-                    }
-                )
-            }
-            if (it.attachActionEventMapper) {
-                configBuilder.setRumActionEventMapper(
-                    object : EventMapper<ActionEvent> {
-                        override fun map(event: ActionEvent): ActionEvent? {
-                            return rumPlugin.mapActionEvent(event)
-                        }
-                    }
-                )
-            }
-            if (it.attachResourceEventMapper) {
-                configBuilder.setRumResourceEventMapper(
-                    object : EventMapper<ResourceEvent> {
-                        override fun map(event: ResourceEvent): ResourceEvent? {
-                            return rumPlugin.mapResourceEvent(event)
-                        }
-                    }
-                )
-            }
-            if (it.attachErrorEventMapper) {
-                configBuilder.setRumErrorEventMapper(
-                    object : EventMapper<ErrorEvent> {
-                        override fun map(event: ErrorEvent): ErrorEvent? {
-                            return rumPlugin.mapErrorEvent(event)
-                        }
-                    }
-                )
-            }
-            if (it.attachLongTaskEventMapper) {
-                configBuilder.setRumLongTaskEventMapper(
-                    object : EventMapper<LongTaskEvent> {
-                        override fun map(event: LongTaskEvent): LongTaskEvent? {
-                            return rumPlugin.mapLongTaskEvent(event)
-                        }
-                    }
-                )
-            }
-        }
-    }
+//    private fun attachEventMappers(
```

Comment:
why this is commented out?

Reply:
Event mappers and attach to existing are more complicated. I created new tasks to those specifically. Commenting them out for now to get core functionality in.

Reply:
On Android we usually add TODO marker with a ticket reference, but ok like this.